src/main/java/seedu/address/logic/parser/StarCommandParser.java

```diff
@@ -24,7 +24,7 @@ public class StarCommandParser implements Parser<StarCommand> {
     public StarCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_STAR); // tokenise args by PREFIX_STAR
+                ArgumentTokenizer.tokenize(args, PREFIX_STAR); // tokenize args by PREFIX_STAR
```

Comment:
Just to check, do we standardise to American English spelling?

Reply:
Yeap! I think it was mentioned in the coding standard. Make America great again!